src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -360,8 +359,12 @@ public String save() {
 
 
             return redirectPage + (!redirectPage.contains("?") ? "?" : "&") + "faces-redirect=true";            
-            
-        } else {
+
+        //Happens if user is logged out while editing
+        } else if (!session.getUser().isAuthenticated()) {
```

Comment:
Just tested and I suppose it is worth noting that you can log out in another tab and still change the password, but, as Oscar mentioned you'll already need to know the user's password at that point. 

What's weird is that logging out in another tab and then saving changes (with correct password) does not renew the session / log the user in-- BUT the password _does_ update for the account. The user just has to log back in afterwards. This isn't a security hazard but is kind of wonky.